src/toast/ops/scan_wcs.py

```diff
@@ -0,0 +1,344 @@
+# Copyright (c) 2015-2020 by the parties listed in the AUTHORS file.
+# All rights reserved.  Use of this source code is governed by
+# a BSD-style license that can be found in the LICENSE file.
+
+import numpy as np
+import traitlets
+
+from ..observation import default_values as defaults
+from ..pixels import PixelData, PixelDistribution
+from ..pixels_io_wcs import read_wcs_fits
+from ..timing import function_timer
+from ..traits import Bool, Instance, Int, Unicode, trait_docs
+from ..utils import Logger
+from .operator import Operator
+from .pipeline import Pipeline
+from .pointing import BuildPixelDistribution
+from .scan_map import ScanMap, ScanMask
+
+
+@trait_docs
+class ScanWCSMap(Operator):
```

Comment:
Is there no way to write a `ScanMap` operator that works on any `Pixels` class?

Reply:
I agree we should generalize this.  Captured now in #574.